docs-starlight/src/content/docs/03-features/07-scaffold.md

```diff
@@ -32,7 +32,7 @@ This will create a `terragrunt.hcl` in your current working directory, with roug
 ```hcl
 # terragrunt.hcl
 
-# This is a Terragrunt module generated by boilerplate.
+# This is a Terragrunt unit generated by Gruntwork Boilerplate (https://github.com/gruntwork-io/boilerplate).
```

Comment:
We shouldn't edit this without editing this:
https://github.com/gruntwork-io/terragrunt/blob/main/cli/commands/scaffold/scaffold.go#L57

Reply:
Updated!